Documents/UseCasesForWebsite.txt

```diff
@@ -1,4 +1,12 @@
--  user wants to see the info page 
--  user wants to apply as artist 
+User 
+
+- user wants to see the info page 
```

Comment:
Very redundant. It should be as simple as: View info page, Donate, RequestProstetic

Reply:
Also, divide it into subsystems:
Website, mobile app.